javascripts/buzz.js

```diff
@@ -224,6 +224,10 @@ app.buzz = {
 
           app.buzz.infiniteScrollCalled = true;
           var from = $('.buzz-item').length + 1;
```

Comment:
Yeah, it just delays the problem by adding the if length == 4

On Thu, Feb 26, 2015 at 12:34 AM, Paul Robinson <notifications@github.com>
wrote:

> In javascripts/buzz.js
> <https://github.com/jboss-developer/www.jboss.org/pull/804#discussion_r25411431>
> :
>
> > @@ -224,6 +224,10 @@ app.buzz = {
> >
> >            app.buzz.infiniteScrollCalled = true;
> >            var from = $('.buzz-item').length + 1;
>
> I think the real fix is to change this to:
>
> var from = $('.buzz-item').length
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-developer/www.jboss.org/pull/804/files#r25411431>
> .
>